### PR TITLE
docs: add kratos quickstart config notes

### PR DESCRIPTION
**Pull request by vinckr**
This adds a small section pointing out the quickstart configuration, where it can be found and additional pointers, as we get questions on that frequently. 

Works fine locally now!

### CODE REVIEW

File: docs/docs/quickstart.mdx [hunk 1]
```diff
@@ -630,3 +630,19 @@ immediately signed in after registration.
...
+In this tutorial we use a simplified configuration.  
+You can find it in
+[`contrib/quickstart/kratos/email-password/kratos.yml`](https://github.com/ory/kratos/blob/master/contrib/quickstart/kratos/email-password/kratos.yml).
+In the same place the
+[identity schema](https://github.com/ory/kratos/blob/master/contrib/quickstart/kratos/email-password/identity.schema.json)
+used in the tutorial can be found. The configuration gets loaded in
+docker-compose as specified in the
+[`quickstart.yml`](https://github.com/ory/kratos/blob/master/quickstart.yml).
+
+<CodeFromRemote src="https://github.com/ory/kratos/blob/master/contrib/quickstart/kratos/email-password/kratos.yml" />
```
Comment (aeneasr) [inline docs/docs/quickstart.mdx:645]: Usage is not correct, here's an example:

```
<CodeFromRemote
  lang="js"
  link="https://github.com/ory/kratos-selfservice-ui-node/blob/master/src/middleware/simple.ts"
  src="https://raw.githubusercontent.com/ory/kratos-selfservice-ui-node/master/src/middleware/simple.ts"
/>
```

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: docs/docs/quickstart.mdx (hunk 1 above)
Comment (vinckr) [reply docs/docs/quickstart.mdx]: Thanks for the heads up, will fix this in hydra/docs as well

File: docs/versioned_docs/version-v0.6/quickstart.mdx [hunk 2]
```diff
@@ -630,3 +630,22 @@ immediately signed in after registration.
...
+In the same place the
+[identity schema](https://github.com/ory/kratos/blob/master/contrib/quickstart/kratos/email-password/identity.schema.json)
+used in the tutorial can be found. The configuration gets loaded in
+docker-compose as specified in the
+[`quickstart.yml`](https://github.com/ory/kratos/blob/master/quickstart.yml).
+
+<CodeFromRemote 
+  lang="yml"
+  link="https://github.com/ory/kratos/blob/master/quickstart.yml"
+src="https://github.com/ory/kratos/blob/master/contrib/quickstart/kratos/email-password/kratos.yml" />
```
Comment (aeneasr) [inline docs/versioned_docs/version-v0.6/quickstart.mdx:648]: Please run `cd docs; npm run format`

**Comment by vinckr** [comment]
Same as with the hydra PR, ended up manually formatting it. 

qs2

**Pull request merged**